front/app/api/project_reviews/useApproveProject.ts

```diff
@@ -0,0 +1,27 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+
+import fetcher from 'utils/cl-react-query/fetcher';
+
+import projectReviewKeys from './keys';
+import { ProjectReview } from './types';
+
+const approveProjectReview = async (projectId: string) =>
+  fetcher<ProjectReview>({
+    path: `/projects/${projectId}/review`,
+    action: 'patch',
+    body: { project_review: { state: 'approved' } },
+  });
+
+const useApproveProjectReview = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation<ProjectReview, CLErrors, string>({
+    mutationFn: approveProjectReview,
+    onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: projectReviewKeys.items() });
```

Comment:
Is it possible to only invalidate the specific project item here, rather than all items?